fpga/mcs.wake

```diff
@@ -29,9 +29,15 @@ global target makeBitstream plan =
     def runDir = outputDir
 
     def ffile =
-      map (relative runDir _.getPathName) vsrcs
-      | catWith "\n"
-      | write "{runDir}/verilog.F"
+      def vnames =
+        vsrcs
+        | map (relative runDir _.getPathName)
+        | catWith "\n"
+      def enames =
+        getenv "WAKE_IP_RESOURCE_FILES"
+        | getOrElse ""
+        | replace `:` "\n"
```

Comment:
wouldn't this always give `""` unless you have `WAKE_IP_RESOURCE_FILES` set in your actual environment? `getenv` grabs the value of `WAKE_IP_RESOURCE_FILES` from the user environment, not the environment that the job is actually running in.

Reply:
Oh fuck, that was the reason it is done by script?

I thought wake couldnt read env vars at all, and Wes told me it could so I went ahead with this...

Reply:
Since this won't work as initially intended, should we close out this PR?

Reply:
yeah